Assets/Scripts/Player/PlayerHandDetection.cs

```diff
@@ -57,15 +57,16 @@ public class PlayerHandDetection : MonoBehaviour {
 		foreach (Interactable interactable in interactableList) {
 			float distance = (transform.position - interactable.transform.position).sqrMagnitude;
 
-			if (distance < lowestDistance) {
+			if (distance < lowestDistance || interactable.CompareTag(priorityInteractableTag)) {
```

Comment:
Don't check priority if no tag is selected.
```suggestion
			if (distance < lowestDistance || (!string.IsNullOrEmpty(priorityInteractableTag) && interactable.CompareTag(priorityInteractableTag))) {
```